xml/schema/Core/ActionSchedule.xml

```diff
@@ -347,4 +347,14 @@
     <comment>Used for multilingual installation</comment>
     <add>4.7</add>
   </field>
+  <field>
+    <name>created_date</name>
```

Comment:
Would it be more standard to put in a <title> field of 'Created on' or better yet 'Created Date' rather than obscure the field label in code at https://github.com/civicrm/civicrm-core/pull/18298/files#diff-e179caac63281aad4b5df3aab2e370e2R161 and https://github.com/civicrm/civicrm-core/pull/18298/files#diff-2b6f0ba5bef33e7e8e29da6c1b6e29e0R22